source/client/HandView.py

```diff
@@ -48,32 +58,19 @@ def __init__(self, controller, display):
             self.top_discard_wrapped = UICardWrapper(self.top_discard, (100, 25), UIC.scale)
```

Comment:
Do we need to set this here? ButtonDisplay does this wrapping when there are discarded cards, and there aren't any during init.

Reply:
Removed all but initialization of hand_view.pickup_pile_sz.